pandas/tests/test_resample.py

```diff
@@ -3300,3 +3300,20 @@ def test_aggregate_with_nat(self):
 
             # if NaT is included, 'var', 'std', 'mean', 'first','last'
             # and 'nth' doesn't work yet
+
+    # Issue: 17530
```

Comment:
move the comment inside the function, move to pandas/tests/groupy/test_timegrouper.py